src/test/java/com/agnieszkapawska/flashcards/FlashcardAndQuestionTagAbstractTests.java

```diff
@@ -0,0 +1,45 @@
+package com.agnieszkapawska.flashcards;
+
+import com.agnieszkapawska.flashcards.domain.dtos.FlashcardSaveDto;
+import com.agnieszkapawska.flashcards.domain.dtos.FlashcardSaveResponseDto;
+import com.agnieszkapawska.flashcards.domain.models.Flashcard;
+import com.agnieszkapawska.flashcards.domain.models.QuestionTag;
+import org.junit.Before;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Set;
+
+public abstract class FlashcardAndQuestionTagAbstractTests extends FlashcardsApplicationAbstractTests {
```

Comment:
So you have this class only to have those properties? At least in our project we used to create another class like `HelpersFactory` which has static methods for this. And then in tests we call `HelpersFactory.sampleFlashcardSaveDto()`

Reply:
Good to know ! :)